src/defines/define.ts

```diff
@@ -0,0 +1,10 @@
+import {MagickFormat} from '../magick-format';
+
+export interface IDefine {
```

Comment:
Thanks for making the decision to add an interface here. After merging your PR I will be adding a lot more.